### DISCUSSION TITLE
Suggestions: Show only master volume. Use legacy mixer by default

### DISCUSSION THREAD

**Discussion by pep0w** [File-New-Project/EarTrumpet#1255] [Ideas]
Since all programs that make a sound are listed with their own volume control, the volume popup can get huge very quickly,
Showing only the master volume slider would solve this.

Another thing I would suggest is to have an option to use the legacy mixer by default. I would like to save time by not having to click many times to get to it.
And before anyone says that I can just use the other volume mixer I'd like to point out that that design also suffers from the same problem I mentioned in the first suggestion. It's basically unusable.

**Comment by Parsnip** [comment]
I also wouldn't mind an option to hide everything but the master volume.
